services_app/src/main/java/org/opendatakit/services/sync/service/logic/AggregateSynchronizer.java

```diff
@@ -1053,14 +1053,20 @@ public void uploadInstanceFileBatch(List<CommonFileAttachmentTerms> batch,
   @Override
   public void downloadInstanceFileBatch(List<CommonFileAttachmentTerms> filesToDownload,
       String serverInstanceFileUri, String instanceId, String tableId) throws HttpClientWebException, IOException {
-    // boolean downloadedAllFiles = true;
+    downloadInstanceFileBatch(filesToDownload, serverInstanceFileUri, instanceId, tableId, false);
+  }
 
+  @Override
+  public void downloadInstanceFileBatch(List<CommonFileAttachmentTerms> filesToDownload,
+      String serverInstanceFileUri, String instanceId, String tableId, boolean reduce) throws HttpClientWebException, IOException {
+    // boolean downloadedAllFiles = true;
     URI instanceFilesDownloadUri = wrapper.constructInstanceFileBulkDownloadUri(serverInstanceFileUri, instanceId);
 
     ArrayList<OdkTablesFileManifestEntry> entries = new ArrayList<OdkTablesFileManifestEntry>();
     for (CommonFileAttachmentTerms cat : filesToDownload) {
       OdkTablesFileManifestEntry entry = new OdkTablesFileManifestEntry();
       entry.filename = cat.rowPathUri;
+      entry.reduceFileSize = reduce ? "t" : "f";
```

Comment:
I strongly recommend to use "true" and "false" instead of "t" and "f" as you can then simply use the Boolean class to convert the parameters. Consistency is important and if it's meant as true and false then use true and false.